packages/langium/src/workspace/documents.ts

```diff
@@ -327,6 +404,17 @@ export class DefaultLangiumDocuments implements LangiumDocuments {
     }
 
     invalidateDocument(uri: URI): void {
```

Comment:
Idea: return `LangiumDocument | undefined` so it's easier to customize in subclasses (same for `deleteDocument`).

Reply:
Good point!

Reply:
Changed it.